RNTester/js/Shared/TextInlineView.js

```diff
@@ -0,0 +1,205 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+const React = require('React');
```

Comment:
```suggestion
const React = require('react');
```
As part of a migration from Haste to standard imports, `React` (the Haste name) has been replaced with `react`.